modules/cluster-autoscaler-config-priority-expander.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * machine_management/applying-autoscaling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cluster-autoscaler-config-priority-expander_{context}"]
+= Configuring a priority expander for the cluster autoscaler
+
+When the cluster autoscaler uses the priority expander, it scales up by using the node group with the highest user-assigned priority.
+To use this expander, you must create a config map that defines the priority of your node groups.
+
+For each specified priority level, you must create regular expressions to identify machine sets that you want to use when prioritizing a node group for selection.
+The regular expressions must match the name of any compute machine set that you want the cluster autoscaler to consider for selection.
+
+.Prerequisites
+
+* You have deployed an {product-title} cluster that uses the Machine API.
+* You have access to the cluster using an account with `cluster-admin` permissions.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. List the compute machine sets on your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machines.machine.openshift.io
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                        DESIRED   CURRENT   READY   AVAILABLE   AGE
+archive-agl030519-vplxk-worker-us-east-1c   1         1         1       1           25m
+fast-01-agl030519-vplxk-worker-us-east-1a   1         1         1       1           55m
+fast-02-agl030519-vplxk-worker-us-east-1a   1         1         1       1           55m
+fast-03-agl030519-vplxk-worker-us-east-1b   1         1         1       1           55m
+fast-04-agl030519-vplxk-worker-us-east-1b   1         1         1       1           55m
+prod-01-agl030519-vplxk-worker-us-east-1a   1         1         1       1           33m
+prod-02-agl030519-vplxk-worker-us-east-1c   1         1         1       1           33m
+----
+
+. For each compute machine set that you want to reference, create a regular expression that matches its name.
+
+. Create a `cluster-autoscaler-priority-expander.yml` YAML file that defines a config map similar to the following:
++
+--
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-autoscaler-priority-expander # <1>
+  namespace: kube-system # <2>
+data:
+  priorities: |- # <3>
+    10:
+      - *fast*
+      - *archive*
+    40:
+      - *prod*
+----
+<1> You must name config map `cluster-autoscaler-priority-expander`.
+<2> You must create the config map in the `kube-system` namespace.
```

Comment:
According to [this](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/expander/priority/readme.md#configuration ) should be in openshift-machine-api namespace

Reply:
Excellent catch, thanks!